src/crimson/os/seastore/cache.cc

```diff
@@ -989,7 +989,8 @@ record_t Cache::prepare_record(
     } else {
       auto sseq = NULL_SEG_SEQ;
       auto stype = segment_type_t::NULL_SEG;
-      if (cleaner != nullptr) {
+      if (cleaner != nullptr && i->get_paddr().get_addr_type() ==
```

Comment:
This isn't handled correctly for rbm addresses yet.  Just do assert(i->get_paddr().get_addr_type() == addr_types_t::SEGMENT)

Reply:
Since `i` is a member of mutual_block_list, i->get_paddr() here can be the rbm address.
Consequently, adding assert will trigger assert during unittest-transaction-manager.